src/router.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import AppWithState from '../src/Links/states';
+import AppWithRedux from '../src/Links/redux';
+import NavBar from '../src/Components/Navbar';
+import HomePage from './Links/home';
+
+
+class Router extends Component {
```

Comment:
Router name is very confusing, pls use App, or RouteApp, or something like that

Reply:
and if it's a stateless component, then it should be functional instead of class based component